pages/common/sed.md

```diff
@@ -2,21 +2,21 @@
 
 > Run replacements based on regular expressions
 
-- replace all occurrences of a string in a file, and print the result
+- replace the first occurrence of a string in a file, and print the result
 
-`sed 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed 's/{{find}}/{{replace}}/' {{filename}}`
 
-- replace all occurrences of a string in a file, and overwrite the file
+- replace all occurrences of a string in a file, overwriting the file
   contents
 
-`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed --in-place 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
I usually prefer integrating the option name in the command description, so that the short option is (at least theoretically) decipherable. This is what I did in the command using `-e`, for instance.

For the command using `-i`, however, I thought that trick could result in an awkward phrasing, using both "replace" and "in-place".

Perhaps "replace all occurrences of a string in a file, overwriting the file (i.e. in-place)" would work?